svgdigitizer/svgdigitizer.py

```diff
@@ -11,6 +11,7 @@
 ref_point_regex_str = r'^(?P<point>(x|y)\d)\: ?(?P<value>-?\d+\.?\d*) ?(?P<unit>.+)?'
 scale_bar_regex_str = r'^(?P<axis>x|y)(_scale_bar|sb)\: ?(?P<value>-?\d+\.?\d*) ?(?P<unit>.+)?'
 scaling_factor_regex_str = r'^(?P<axis>x|y)(_scaling_factor|sf)\: (?P<value>-?\d+\.?\d*)'
+curve_regex_str = r'^curve: ?(?P<curve_id>.*)'
```

Comment:
We could consider putting all these regexes into a dict, say `label_patterns`. Then we could further down also check that all labels of paths match exactly one of these regular expressions and warn otherwise.

Reply:
In fact, `labeled_paths` could then already return a dict of the form
```
{
  "ref_point": [(text,path), (text,path)]
}
```
and all the regex handling would be confined to `labeled_paths` and we do not need to repeat it in several places.
```

Reply:
Have you thought of something like in `try with label_pattern dict`?

Reply:
I don't understand the question, sorry.

Reply:
In the commit `try with label_pattern dict` I tried to implement the idea you mentioned above.
Now the question is if you think the implementation fits your idea?